code/modules/research/xenobiology/crossbreeding/consuming.dm

```diff
@@ -136,7 +136,7 @@ Consuming extracts:
 	name = "water cookie"
 	desc = "A transparent blue cookie. Constantly dripping wet."
 	icon_state = "blue"
-	taste = /datum/reagent/water
+	taste = "water"
```

Comment:
```suggestion
	taste = "/datum/reagent/water"
```

Reply:
![image](https://user-images.githubusercontent.com/44720187/163201794-9a2846d1-acfe-48d2-94df-83aa301b3786.png)

Reply:
THE !$!# TASTE OF /datum/reagent/water!